src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/JsonTypeConverters.kt

```diff
@@ -0,0 +1,60 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.domain
+
+import com.beust.klaxon.Converter
+import com.beust.klaxon.JsonValue
+import com.beust.klaxon.KlaxonException
+import uk.gov.justice.digital.hmpps.pecs.jpc.domain.price.Supplier
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+/**
+ * Contains common shared JSON type converters for converting to domain entity field types.
+ */
+
+@Target(AnnotationTarget.FIELD)
+annotation class JsonDateConverter
```

Comment:
These were all in separate before files and no real need to be.  The naming needed some attention also.